pep-0458.txt

```diff
@@ -504,22 +491,110 @@ MUST be offline and independent of other keys.  For simplicity of key
 management, without sacrificing security, it is RECOMMENDED that the keys of
 the *targets* role be permanently discarded as soon as they have been created
 and used to sign for the role.  Therefore, the *targets* role SHOULD require
-(1, 1) keys.  Again, this is because the keys are going to be permanently
+(2, 3) keys.  Again, this is because the keys are going to be permanently
```

Comment:
If we're discarding, why 2,3?

Reply:
So that a key generation weakness is hopefully not fatal

Reply:
How does 2,3 help more than 2,2 or 3,3 if we're tossing the keys?  I'm not sure I understand the threshold rationale.  Should we clarify / change this?  I'm okay with 1,1 or 2,2 or really anything given this is PEP 458.

Reply:
Ok, I see the point of contention now. Yes, (2, 2) should be reasonable.